datajoint/connection.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, host, user, passwd, init_fun=None):
             port = int(port)
         else:
             port = config['database.port']
-        self.conn_info = dict(host=host, port=port, user=user, passwd=passwd)
+        self.conn_info = dict(host=host, port=port, user=user, passwd=passwd, max_allowed_packet=1024**3) # 1073741824
```

Comment:
When I encountered the broken pipe issues in `pymysql`, I found that `pymysql` has its own `max_allowed_packet` for some reason that does not reflect the servers `max_allowed_packet`. Therefore I set it to the maximally allowed value by MySQL to let the server settings decide which packet sizes are accepted. Alternatively, a cleaner solution would be to put that value in `dj.config`.